src/paddle.ts

```diff
@@ -1,7 +1,25 @@
+import type { DataSource } from 'typeorm';
+import { User } from './entity';
+import { queryReadReplica } from './common/queryReadReplica';
+
 export enum SubscriptionCycles {
   Monthly = 'monthly',
   Yearly = 'yearly',
 }
 
 export const isPlusMember = (cycle: SubscriptionCycles | undefined): boolean =>
   !!cycle?.length || false;
+
+export const isUserPlusMember = async (
+  con: DataSource,
+  userId?: string,
+): Promise<boolean> => {
+  const { subscriptionFlags } = await queryReadReplica(con, ({ queryRunner }) =>
+    queryRunner.manager.getRepository(User).findOneOrFail({
+      where: { id: userId },
+      select: ['subscriptionFlags'],
+    }),
+  );
```

Comment:
use props syntax `{ con, userId }`

Reply:
I might change this to use the isPlus from context I added in #2503

Reply:
That would be even better, nice 👌

Reply:
I'll merge this one first, then in #2503 I can it with isPlus from context

Reply:
Changed in 6e69905 in #2503